lang/ghc/Portfile

```diff
@@ -27,11 +26,9 @@ long_description    \
 
 homepage        http://haskell.org/${name}
 
-variant bootstrap \
-	    description {Bootstrap a build of ghc.} {}
-
-# make bootstrapping the default when/if it works
-default_variants    +bootstrap
+# Bootstrapped ghc is the default build
+variant nobootstrap \
```

Comment:
Ugh. I kinda don't want to have to ask you to touch this again now that you've refactored it, but we made a conscious decision a while ago to get rid of all variants that start with "no" and just make them default variants instead, so what you previously had was actually more fitting to how we usually do things.

Reply:
I’ll just rename the variant to `prebuilt`.

Reply:
I personally didn’t like having the `+bootstrap` for the default and nothing said when it’s just the prebuilt external binary, especially when we all want a bootstrap.

Reply:
I agree it makes more sense for the variant to be used for the ‘prebuilt’ option, and renaming it as you suggests works for me.

Reply:
Variant name changed.